src/templates/static/_layout/head.jsx

```diff
@@ -28,6 +28,7 @@ const Head = () => (
 
         <link rel='manifest' href={`${it.root_url}manifest.json`} />
         <script type='text/javascript' src={`${it.root_url}pushwoosh-web-notifications.js`} async />
+        <script type='text/javascript' src='//widget.trustpilot.com/bootstrap/v5/tp.widget.bootstrap.min.js' async />
```

Comment:
is this the correct link? why is it starting with `//`?

Reply:
yes, this is a "protocol-relative" link. It uses http or https depending on what was used to load the current page, https://stackoverflow.com/questions/550038/is-it-valid-to-replace-http-with-in-a-script-src-http

Reply:
we should enforce https, no?

Reply:
added https